components/home/index.js

```diff
@@ -37,12 +37,36 @@ class Home extends Component {
       currentBuildingAddress: '',
       showDirectionsMenu: false,
       showCampusToggle: false,
-      showSuggestionsList: false
+      showSuggestionsList: false,
+      showBack: true
     };
     this.interiorModeOn = this.interiorModeOn.bind(this);
     this.interiorModeOff = this.interiorModeOff.bind(this);
+    this.getADirections = this.getADirections.bind(this);
```

Comment:
can we rename this to something like getCalDirections()

Reply:
@eufekt yea sure it makes more sense to name it 'getCalDirections'